server/server.go

```diff
@@ -20,12 +20,26 @@ type cribbageServer struct {
 func (cs *cribbageServer) Serve() {
 	router := gin.Default()
 
+	router.LoadHTMLGlob("templates/*")
```

Comment:
This is pretty neat. It allows us to template-ize our html pages. This probably isn't a long-term solution (likely we'll want to replace with react or something pretty), but this is a cool thing that gin gives us for "free"

Reply:
🌵 📖 we use backticks here, right?